gateway-ha/src/main/java/io/trino/gateway/ha/util/Constants.java

```diff
@@ -26,6 +13,4 @@ public class QueryIdCachingProxyHandler
     public static final String OAUTH_PATH = "/oauth2";
     public static final String AUTHORIZATION = "Authorization";
     public static final String USER_HEADER = "X-Trino-User";
-
-    private QueryIdCachingProxyHandler() {}
```

Comment:
Please don't remove this constructor.

Reply:
you mean we have to retain QueryIdCachingProxyHandler class file with this constructor?

Reply:
No, I meant keeping a private constructor.